source/deployment-guide/mobile/configure-microsoft-intune-mam.rst

```diff
@@ -0,0 +1,398 @@
+
+Identity Enforcement by Authentication Method
+---------------------------------------------
+
+Apply the same identity rule everywhere.
+
+OAuth / Entra (Mobile)
+~~~~~~~~~~~~~~~~~~~~~~
+
+* Only the access token is used.
+* The ``oid`` claim is required.
+
+SAML (Web Login)
+~~~~~~~~~~~~~~~~~
+
+* ``SamlSettings.IdAttribute`` must map to ``objectidentifier``.
+* Email, UPN, and ``immutableID`` are not supported.
+
+LDAP (Entra ID Domain Services)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Use ``msDS-aadObjectId`` as the identity attribute.
+* Do not use ``objectGUID``.
+
+Mandatory Microsoft Entra API Permissions
+-----------------------------------------
+
+The following Microsoft Entra API permissions are mandatory for Intune MAM enrollment and server-side token validation. Missing permissions will block enrollment.
+
+Grant admin consent for each required permission below:
+
++-----------------------+--------------------------------------------------------------+
+| Component             | Permission                                                   |
++=======================+==============================================================+
+| Mattermost Mobile App | Microsoft Mobile Application Management →                    |
+|                       | ``user_impersonation`` (Delegated)                           |
++-----------------------+--------------------------------------------------------------+
+| Mattermost Server     | ``https://msmamservice.api.application/.default``            |
++-----------------------+--------------------------------------------------------------+
+
+.. note::
+
+    ``user_impersonation`` appears as ``DeviceManagementManagedApps.ReadWrite``.
+
+With prerequisites met and identity requirements understood, proceed to the configuration steps in the next section.
+
+Required Mattermost Server Configuration
+----------------------------------------
+
+1. Go to **System Console > Environment > Mobile App Management**.
+2. Enable **Microsoft Intune App Protection Policies**.
+3. Set ``IdAttribute`` to ``objectId``.
+4. Save your changes.
+
+Validation Checklist
+--------------------
+
+Before rolling out to production, validate the configuration using a test user account. Confirm the following values match for the same user:
+
+* Azure AD ``objectId``
+* MSAL access token ``oid`` claim
+* SAML ``objectidentifier`` (if applicable)
+* LDAP ``msDS-aadObjectId`` (if applicable)
+
+Any mismatch will cause Intune MAM enrollment to fail.
+
+Deploy or Update Mattermost Mobile Apps
+---------------------------------------
+
+Install the latest Mattermost iOS mobile app using one of the following methods:
+
+* Public App Store
+* Microsoft Intune deployment
+* Private or wrapped builds
+
+.. note::
+
+   Mattermost Beta (``com.mattermost.rnbeta``) and Production (``com.mattermost.rn``) apps require **separate** Microsoft Entra app registrations.
```

Comment:
this is not true either, both can be configured with the same Entra app in the Expose an API app registration